src/irmin-pack/unix/file_manager.ml

```diff
@@ -206,7 +206,8 @@ struct
     let open Result_syntax in
     if generation = 0 then Ok None
     else
-      let* m = Mapping_file.open_map ~root ~generation in
+      let path = Irmin_pack.Layout.V3.mapping ~generation ~root in
```

Comment:
All these should use `V4`, just for consistency.

(Future PR: I think we ought to have a `Latest` layout, similar to what we do for the control file, to make these a bit less of a burden to maintain)